lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -183,15 +183,15 @@ def store(payment_method, options = {})
           if options[:customer]
             customer_id = options[:customer]
           else
-            post[:validate] = options[:validate] unless options[:validate].nil?
             post[:description] = options[:description] if options[:description]
             post[:email] = options[:email] if options[:email]
             options = format_idempotency_key(options, 'customer')
             customer = commit(:post, 'customers', post, options)
             customer_id = customer.params['id']
           end
+          validate = !options[:validate].nil? ? options[:validate] : false
```

Comment:
Nitpick, but Ruby norms prefer not using '!' when possible, and here we can use
`options[:validate].present? ? options[:validate] : false`

Reply:
However, we might want to go with some extra resilience by enforcing `true`/`false`, via something like:
`validate = (options[:validate] == "true").to_s`

Reply:
Also want to confirm that not validating should be the default case? That seems to be true of the initial implementation but it's something we should be sure about.

Reply:
The ticket description sounded to me as we want to set to 'false' by default but  now you mentioned I'm not sure and asked there to clarify.

Reply:
@curiousepic  I think it makes sense to pass `validate` parameter to /attach only if this option was requested by user. Otherwise, we should let Stripe to perform their default actions (currently they do validation by default). So, I made a change to pass a `validate` parameter only if it was provided in `options`.